src/algorithm/joint-configuration.hpp

```diff
@@ -345,7 +343,109 @@ namespace pinocchio
   {
     dIntegrate<LieGroupMap,Scalar,Options,JointCollectionTpl,ConfigVectorType,TangentVectorType,JacobianMatrixType>(model, q.derived(), v.derived(), PINOCCHIO_EIGEN_CONST_CAST(JacobianMatrixType,J),arg,op);
   }
-  
+
+  /**
+   *
+   * @brief   Transport a matrix from the terminal to the originate tangent space of the integrate operation.
+   *
+   * @details This function performs the parallel transportation of an input matrix whose columns are expressed in the tangent space of the integrated element \f$ q \oplus v \f$,
+   *          to the tangent space of \f$ q \f$.
```

Comment:
The terminology is *the tangent space __at__ \f$ q \f$ __of__ manifold ...*.

May be it would be good to explain this in term of velocity expressed in different frames.

Reply:
Done.

Reply:
The term *parallel transportation* is a term of differential geometry, which most users won't understand. I have myself only a vague notion of what it is. From the documentation, I am not able to know what the function do.

Reply:
I've updated the doc. Please provide additional suggestions to make it better.

Reply:
FYI: doxygen command \copydoc can avoid duplication.

For SE(3), it corresponds to (*something like*) transforming a velocity expressed in q+v into q. As it is, the documentation is already much better.

Reply:
Yep.